### ISSUE TITLE
Float properties from XML discard fractional part

### ISSUE THREAD

**Issue by not-surt** [ianharrigan/haxeui#273]
On Windows with Neko and CPP.

Floating point values in properties from XML layouts discard the fractional part.
eg. "7.75" and "2.5" in XML result in 7 and 2 when run.

main.xml
```xml
<?xml version="1.0" encoding="utf-8" ?>
<hslider id="slider" min="0" max="15" pos="7.75" incrementSize="2.5" width="100%" />
```

Main.hx
```haxe
package;

import haxe.ui.toolkit.controls.HSlider;
import haxe.ui.toolkit.core.interfaces.IDisplayObject;
import haxe.ui.toolkit.core.Root;
import haxe.ui.toolkit.core.Toolkit;

class Main {
	private function new():Void {
	}

	public static function main():Void {
		Toolkit.init();
		var main = new Main();
		Toolkit.openFullscreen(function(root:Root) {
			var view:IDisplayObject = Toolkit.processXmlResource("assets/main.xml");
			root.addChild(view);
			var slider:HSlider = root.findChild("slider", HSlider);
			trace(slider.pos);
			trace(slider.incrementSize);
		});
	}
}
```

**Comment by ianharrigan** [comment]
This should be fixed now in 1.7.15 in haxelib, can you close this when you get a moment.

Thanks!

Ian

**Comment by not-surt** [comment]
Looks to be working fine now, cheers!
